shell/plugins/vee-validate.js

```diff
@@ -0,0 +1,117 @@
+import { extend as veeExtend } from 'vee-validate';
+import * as veeRules from 'vee-validate/dist/rules';
+import formRulesGenerator from '@shell/utils/validators/formRules/index';
+import CustomValidators from '@shell/utils/custom-validators';
+
+export function extend(rule) {
```

Comment:
I am not so sure about exporting a function called `extend` or `create` is a good idea.
It may create overlapping issues and be unclear.

Is it needed to be exported?